docs/keto/reference/ory-permission-language.mdx

```diff
@@ -1,14 +1,11 @@
----
-id: ory-permission-language
-title: Ory Permission Language specification
-sidebar_label: Ory Permission Language
----
+# The "Ory Permission Language" Specification
 
 Enforcing fine-grained permissions is a critical building block of mature technology solutions that protect privacy and identity
-in the information age. Several proprietary languages used to represent permission already exist, such as Rego or Casbin. Most
-permissions are defined by developers who are likely familiar with Web technologies like JavaScript or Typescript. There is a need
-for a developer-friendly configuration language for permissions that has a learning curve small enough so that most developers can
-understand and use it with minimal effort. To fulfill this need, we defined the permissions configuration language as a subset of
+in the information age. Several proprietary languages to represent permission already exist, namely the
+[Authzed Schema Language](https://docs.authzed.com/reference/schema-lang) and [Auth0 FGA](https://docs.fga.dev/). Most permissions
+are defined by normal developers who typically are most familiar with Web technologies like JavaScript or Typescript. We see the
+need for a developer-friendly configuration language for permissions that has such a small learning curve that most developers can
+understand and use it with close to no effort. We therefore chose to define our permissions configuration language as a subset of
```

Comment:
```suggestion
are defined by normal developers who typically are most familiar with Web technologies like JavaScript or Typescript. Ory recognizes the need for a configuration language for permissions with a short learning curve that developers can understand and use without effort. Because of this the Ory Permissions Language  is defined as a subset of
```